app/footer/footer.component.ts

```diff
@@ -52,12 +51,12 @@ import {Location} from 'angular2/router';
 
 export class FooterComponent {
 
-  constructor(
-    private location: Location
-  ) {}
+  locationPath(): string {
```

Comment:
For some reason, jasmine's `spyOn` doesn't work on getters.  So if I want to mock it, it has to be an actual function.

This was more straight forward before they took `Location` out of the router.